### ISSUE TITLE
Bridge support

### ISSUE THREAD

**Issue by jesperpedersen** [pgexporter/pgexporter#151]
Add functionality to collect metrics from other Prometheus servers, and include their metrics in the entire result

**Comment by resyfer** [comment]
@jesperpedersen Does this means just directly adding their metrics from `/metrics` (of respective servers) to our pool of metrics?

**Comment by jesperpedersen** [comment]
Yes, download each URL endpoint, strip the HTTP header, sort the download entries together with the local ones (if there are any), and push out.

Easy test would be 2 pgexporters, one with no database attached but linked with the other one that has a database attached.

The `bridge` should be a comma separated list of endpoints

**Comment by jesperpedersen** [comment]
We need to do

```
[pgexporter]
...
bridge=endpoint1:port1,endpoint2:port2,...
bridge_port=5003
```

and then stick the endpoint into the metrics

`mymetric 1` -> `mymetric{endpoint="endpoint1:port1"} 1`
